UserService/Controllers/Health/HealthController.cs

```diff
@@ -0,0 +1,17 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace UserService.Controllers.Health;
+
+[Authorize]

```

Comment:
Are you sure you want to use Authorize here?
It will return 401 unless you provide a valid JWT, I guess our heatlh checks won't have authorization